src/sql/workbench/contrib/dataExplorer/browser/dataExplorer.contribution.ts

```diff
@@ -42,6 +42,16 @@ configurationRegistry.registerConfiguration({
 		'datasource.connectionGroups': {
 			'description': localize('datasource.connectionGroups', "data source groups"),
 			'type': 'array'
+		},
+		'datasource.connectionSort': {
```

Comment:
I know it is not started by you, but why not use the const string defined above?

Reply:
updated, fixed other const string keys as well.